app/reducers/treatment.js

```diff
@@ -0,0 +1,56 @@
+import axios from 'axios'
```

Comment:
Well we already have the plan on the store. What I mean is simply keeping `currentTreatmentId` on the store so we can grab it from the existing plan object. Avoids another round trip/object to keep in sync w/ potential updates